.flake8

```
@@ -1,6 +1,6 @@
 [flake8]
 # B905 should be enabled when we drop support for 3.9
-ignore = E203, E266, E501, W503, B905
+ignore = E203, E266, E501, W503, B905, B028
```

Maybe we should just move forward flake8-bugbear version to 23.1.17 + disable B907
```suggestion
ignore = E203, E266, E501, W503, B905, B907
```

---

flake8-bugbear isn't pinned for us, so I think we're automatically using the latest.

---

Done